apps/aehyperchains/src/aehc_consensus_hyperchains.erl

```diff
@@ -112,8 +112,9 @@
         , deserialize_pos_pow_field/1
         ]).
 
+-include_lib("aecontract/include/aecontract.hrl").
 -include_lib("aecontract/include/hard_forks.hrl").
--include("../../aecore/include/blocks.hrl").
+-include_lib("aecore/include/blocks.hrl").
```

Comment:
Hyperchains is a dependency of aecore in mono-repo app structure:

- aecore 
  - aehyperchains
  - ...

When we link aecore as "include_lib" we make circular dependency inside (take a look into another included apps, they don't use it).

Reply:
@cna0 Please respond, I don't have enough understanding of erlang dependencies to judge here.
@erlmachinedev What exactly would break if such a circular dependency is introduced?

Reply:
@erlmachinedev @cytadela8 as fas I as I know, erlang takes care of circular deps; I don't think `include_lib` or `include` would make a difference regarding that circular deps. Or am I missing something?